### ISSUE TITLE
deploydocs should create and maintain redirect index.html?

### ISSUE THREAD

**Issue by mortenpi** [JuliaDocs/Documenter.jl#937]
Most Documenter setups have an `index.html` file like this

```html
<meta http-equiv="refresh" content="0; url=http://juliadocs.github.io/Documenter.jl/stable"/>
```

in the root of the `gh-pages` branch to redirect to the latest stable version of the docs. Should `deploydocs` just create this file automatically? Possibly with an option to disable this if the user wants to maintain their own `index.html`.

The behavior could be as follows:

* If there are tagged versions, have `index.html` point to `stable/`.
* If there are not tagged version, have it point to `dev/`.

Related: #264

**Comment by fredrikekre** [comment]
Makes sense. I think most packages don't have that file at all though?

**Comment by fredrikekre** [comment]
We can have a Documenter specific header and only modify it if it was indeed generated by Documenter.

**Comment by kellertuer** [comment]
It seems my instance does not (yet?) create the index.html automatically. Do I have to activate that? It would be handy because until now only the stable/ link works – well “only” – that one works perfectly fine and great :)

**Comment by christopher-dG** [comment]
Bump, I would very much like this feature. I've been pasting an `index.html` in manually for a bunch of packages but I'd prefer to have it handled for me, especially if it can properly choose between `/dev` and `/stable`!

**Comment by mortenpi** [comment]
Fixed by #1657.
